addons/radiodirectionfinder/data/US_AUV_Antena.rvmat

```diff
@@ -0,0 +1,90 @@
+class StageTI
+{
+	texture="#(argb,8,8,3)color(0,0,0,0)";
+};
+ambient[]={1,1,1,1};
+diffuse[]={1,1,1,1};
+forcedDiffuse[]={0,0,0,1};
+emmisive[]={0,0,0,1};
+specular[]={0.67843139,0.67843139,0.67843139,0};
+specularPower=52;
+PixelShaderID="Super";
```

Comment:
Does this really need Super if you just use normal and specular?
Why not just use "NormalMapSpecularMap" shader if that's all you need?